pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "tw-source-finder"
+version = "0.0.1"
+license = "MIT license"
+readme = 'README.md'
+description = "super-fast source finder routine using polygon based approach"
+authors = ["Tony Willis <tony.willis.research@gmail.com>"]
+maintainers = [
+    "Tony Willis <tony.willis.research@gmail.com>",
+    "Samuel Twum <samueltwum1@gmail.com>",
+    "Buntu Ngcebetsha <bngcebetsha@sarao.ac.za>",
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+]
+packages = [
+    { include = "tw_source_finder", from = "src" }
+]
+include = [
+    { path = 'tests'},
+]
+
+[tool.poetry.scripts]
+tw-source-list = 'tw_source_finder.scripts.get_simple_source_list:main'
```

Comment:
@twillis449 Not sure if you have a preferred name to be used to evoke the program. It's currently `tw-source-list`. Other options can be:

`tw-generate-source-list -f xyz.FITS -t 6.5`
`tw-generate -f xyz.FITS -t 6.5`
`source-list-generate -f xyz.FITS -t 6.5`
`tw-gen-source-list -f xyz.FITS -t 6.5`

Reply:
@twillis449 I'm going to merge this in for now. We can always update the script name in the PRs to follow.